crates/pop-common/src/sourcing/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub mod binary;
+pub mod errors;
+pub mod sourcing;
```

Comment:
Think this adds unneccessary complexity/duplication - sourcing::sourcing::*. Suggest moving the content of sourcing.rs to sourcing/mod.rs so its easier to find and easier to navigate.